src/controller/java/zap-generated/chip/devicecontroller/ClusterReadMapping.java

```diff
@@ -263,10 +263,9 @@ public Map<String, Map<String, InteractionInfo>> getReadAttributeMap() {
         new InteractionInfo(
             (cluster, callback, commandArguments) -> {
               ((ChipClusters.ScenesCluster) cluster)
-                  .readCurrentGroupAttribute(
-                      (ChipClusters.ScenesCluster.CurrentGroupAttributeCallback) callback);
+                  .readCurrentGroupAttribute((ChipClusters.IntegerAttributeCallback) callback);
             },
-            () -> new ClusterInfoMapping.DelegatedScenesClusterCurrentGroupAttributeCallback(),
+            () -> new ClusterInfoMapping.DelegatedIntegerAttributeCallback(),
```

Comment:
Old code used to be CurrentGroupAttributeCallback and new code is Integer.

it may be ok (group_id is an integer) however old code had it this way ... so I made jinja codegen special case group_id. We may need to special case it as well or remove the jinja special case to make codegen match.

Reply:
The old code used the "group" callback for all 32-bit integers, I expect.  Because the old code's behavior depended on whether it happened to "see" a group-typed or int32u-typed attribute first....

Reply:
Same as above